tests/e2e-cypress/fixtures/html-popup.fixture.html

```diff
@@ -1,6 +1,9 @@
 <div class="chbabskulturgueter htmlpopup-container">
     <div class="htmlpopup-header">
-        <span>I'm a test window, I'm a test window, I'm a test window, I'm a test window, I'm a test window</span>
+        <span
+            >I'm a test window, I'm a test window, I'm a test window, I'm a test window, I'm a test
+            window</span
+        >
```

Comment:
small detail, linter-formatter sometimes do that, but I would prefer that we do not wrap the tag itself, but just after the tag
```
<span>
...
</span>
```
It is then much more readable

Reply:
Prettier has a very good reason for doing this, see https://prettier.io/blog/2018/11/07/1.15.0#whitespace-sensitive-formatting.
In this test fixture case I don't know if this is relevant though ?